Subject: b43: Mask the PHY TX error interrupt

Mask the PHY TX error interrupt. This error triggers for yet unknown
reasons. It's not that often to really harm the transmission, but it _might_
be more often than 1000 times in 15 seconds. The PHY TX error watchdog however
will reset the controller, if it triggers more than 1000 times in 15 seconds.
This might happen under heavy load and thus kill the interface.

So as a workaround we ignore the PHY TX error.


--mb


--- a/drivers/net/wireless/b43/b43.h
+++ b/drivers/net/wireless/b43/b43.h
@@ -422,7 +422,6 @@ enum {
 					 B43_IRQ_ATIM_END | \
 					 B43_IRQ_PMQ | \
 					 B43_IRQ_MAC_TXERR | \
-					 B43_IRQ_PHY_TXERR | \
 					 B43_IRQ_DMA | \
 					 B43_IRQ_TXFIFO_FLUSH_OK | \
 					 B43_IRQ_NOISESAMPLE_OK | \
